docs/changes/units/17355.bugfix.rst

```diff
@@ -0,0 +1,3 @@
+It is now possible to use ``Unit`` to create dimensionless units with a scale
```

Comment:
Maybe start with "Although rarely needed, it is ..."  Or add a sentence at the end, along the lines of "Note that this is rarely needed, but can occur. For instance, one gets a complex scale from ``(u.mag**0.5).decompose()``"

Reply:
Underlying my suggestion was that it helps to tell users whether or not they need to worry about a changelog entry. Here, they should not.

Reply:
I'll approve, leaving it up to you whether you adjust the changelog message to help less experienced users.

Reply:
If some user can't figure out whether rational or complex unit scales are useful for them or not then we can't help them regardless of what we write down.